applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,91 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+DVFS is critical for balancing performance and power consumption, especially in battery-operated devices.
+
+In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when using classic BLE as it is enough to run this mode.
+However, if performing more computation-consuming tasks DVFS module will dynamically change to a higher frequency to increase performance.
+
+The module is only available for nrf54h20 SoC.
```

Comment:
I would suggest to simply state that DVFS feature must be supported by the SoC and add nRF54H20 as an example of SoC that supports it (I prefer to avoid making sections and subsections related to SoCs/boards to avoid maintaining them later). We use no SoC-specific code here too (we just rely on fact that DVFS API is available).